hooks/pre-exit

```diff
@@ -28,17 +28,23 @@ fi
 # resources, which requires multiple invocations of `Pkg.gc` to be effective.
 
 echo "Running Pkg.gc()"
-julia --color=yes -e 'using Pkg; using Dates; VERSION >= v"1.6-" ? Pkg.gc(; collect_delay=Second(0), verbose=true) : Pkg.gc(; collect_delay=Second(30))'
-sleep 1
-julia --color=yes -e 'using Pkg; using Dates; VERSION >= v"1.6-" ? Pkg.gc(; collect_delay=Second(0), verbose=true) : Pkg.gc(; collect_delay=Second(30))'
+INTVERS="$(tr -d '.' <<<"${BUILDKITE_PLUGIN_JULIA_VERSION:0:3}")"
```

Comment:
Isn't it easier to perform these checks in Julia, in case we ever support more named releases (like the proposed stable, beta, etc)?

Reply:
Fair enough.  I'll put this back into Julia.